ui/public/static/img/web-filters.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="612" height="612" version="1.1"><path fill="#fff" d="m376 0c0 1.08 0 2.16 0 3.3 0 38.76-31.42 70.17-70.17 70.17-38.76 0-70.17-31.42-70.17-70.17l0 0c0-1.14 0-2.22 0-3.3L0 0l0 612 612 0 0-612z" style="fill:#fff;opacity:.800000011920929;isolation:isolate"/><path fill="#fff" d="m142.16 329.81 40.56 0 27.69 147.47 0.5 0 33.28-147.47 37.94 0 30.06 149.28 0.59 0 31.56-149.28 39.78 0-51.69 216.69-40.25 0-29.81-147.47-0.78 0-31.91 147.47-41 0zm287.69 0 63.94 0 63.5 216.69-41.84 0-13.81-48.22-72.84 0-10.66 48.22-40.75 0zm24.34 53.41-17.69 79.5 55.06 0-20.31-79.5z"/></svg>
```

Comment:
@Chadha93 There are 3 #fff color codes in this `.svg` file.
Could you change the 3rd one to `#233239` color code, this will make the WA characters in the `.svg` to be in Meshery Color

The code,

![image](https://user-images.githubusercontent.com/23498248/119567592-1e0acd80-bdca-11eb-8d84-a00409843348.png)

Result: Other parts of the .svg will remain white
![image](https://user-images.githubusercontent.com/23498248/119567646-2f53da00-bdca-11eb-923c-7acedc993d9e.png)